frontend/src/hooks/useCreateMeeting/useCreateMeeting.ts

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+
+import useInput from '@hooks/useInput/useInput';
+import useTimeRangeDropdown from '@hooks/useTimeRangeDropdown/useTimeRangeDropdown';
+
+import { FIELD_DESCRIPTIONS, INPUT_FIELD_PATTERN } from '@constants/inputFields';
+
+const useCreateMeeting = () => {
+  const meetingNameInput = useInput({
+    pattern: INPUT_FIELD_PATTERN.meetingName,
+    errorMessage: FIELD_DESCRIPTIONS.meetingName,
+  });
+  const isMeetingNameInvalid =
+    meetingNameInput.errorMessage !== null || meetingNameInput.value.length < 1;
+
+  const hostNickNameInput = useInput({
+    pattern: INPUT_FIELD_PATTERN.nickname,
+    errorMessage: FIELD_DESCRIPTIONS.nickname,
+  });
+
+  const hostPasswordInput = useInput({
+    pattern: INPUT_FIELD_PATTERN.password,
+    errorMessage: FIELD_DESCRIPTIONS.password,
+  });
+  const isHostInfoInValid =
+    hostNickNameInput.value.length < 1 ||
+    hostPasswordInput.value.length < 1 ||
+    hostNickNameInput.errorMessage !== null ||
+    hostPasswordInput.errorMessage !== null;
+
+  const [selectedDates, setSelectedDates] = useState<string[]>([]);
+  const areDatesUnselected = selectedDates.length < 1;
+  const hasDate = (date: string) => selectedDates.includes(date);
+
+  const handleDateClick = (date: string) => {
+    setSelectedDates((prevDates) =>
+      hasDate(date) ? prevDates.filter((d) => d !== date) : [...prevDates, date],
+    );
+  };
+
+  const { startTime, endTime, handleStartTimeChange, handleEndTimeChange } = useTimeRangeDropdown();
+
+  const isCreateMeetingFormInValid =
```

Comment:
[P2]

invalid 자체가 하나의 단어이기 때문에 아래와 같이 사용하는 것은 어떨까요?
isHostInfoInValid도 마찬가지입니다!

```suggestion
  const isCreateMeetingFormInvalid =
```

Reply:
굿~